api/src/main/java/io/opentelemetry/internal/Utils.java

```diff
@@ -208,4 +208,23 @@ private static String format(String template, @javax.annotation.Nullable Object.
 
     return builder.toString();
   }
+
+  /**
+   * Validates that the array of Strings is 1) even in length, and 2) they can be formed into valid
+   * pairs where the first item in the pair is not null.
+   *
+   * <p>TODO: write unit tests for this method.
+   *
+   * @param keyValuePairs The String[] to validate for correctness.
+   * @throws IllegalArgumentException if any of the preconditions are violated.
+   */
+  public static void validateLabelPairs(String[] keyValuePairs) {
```

Comment:
nit: Do we need to allow null values? null means you don't add the key to the list.

Reply:
not sure. we could ignore null keys, I suppose? Want to create an issue to track it?

Reply:
Let's keep it as it is. We should just create an issue about how to pass Labels and track what we discussed on gitter  and we will address that concern at that point.

Reply:
#1000